src/VisualStudio/Core/Def/Implementation/ProjectSystem/MetadataReferences/VisualStudioMetadataReference.Snapshot.cs

```diff
@@ -82,15 +82,7 @@ private bool SaveMetadataReadingException(Exception e)
 
             protected override DocumentationProvider CreateDocumentationProvider()
             {
-                string xmlDocumentPath;
-                if (ReferencePathUtilities.TryFindXmlDocumentationFile(FilePath, out xmlDocumentPath))
```

Comment:
No and Yes. It isn't using it in the context of an IVsXmlMemberIndexService. It is using it to just shadow copy metadata files along with the help files. It will be incorrect in the case where it'd miss shadow copying an xml help file for a winmd which is placed right next to it, but I'll fix that.

sounds right?